incremental-font-transfer/src/glyph_keyed.rs

```diff
@@ -1,3 +1,4 @@
+use crate::font_patch::PatchingError;
 /// Implementation of Glyph Keyed patch application.
```

Comment:
not sure how this happened, but if these are documenting the overall module they should start with `//!` not `///`.